packages/@ourworldindata/grapher/src/chart/guidedChartUtils.test.ts

```diff
@@ -0,0 +1,84 @@
+import { describe, it, expect } from "vitest"
+import { Url } from "@ourworldindata/utils"
+import {
+    buildArchiveGuidedChartSrc,
+    type ArchiveGuidedChartRegistration,
+} from "./guidedChartUtils.js"
+import { createRef } from "react"
+import { ChartConfigType } from "@ourworldindata/types"
+
+const makeRegistration = (
+    baseUrl: string,
+    defaultQueryParams: Record<string, string | undefined> = {}
+): ArchiveGuidedChartRegistration => ({
+    iframeRef: createRef<HTMLIFrameElement>(),
+    baseUrl,
+    defaultQueryParams,
+    chartConfigType: ChartConfigType.Grapher,
+})
+
+describe(buildArchiveGuidedChartSrc.name, () => {
```

Comment:
nit: you don't need the `.name` here
```suggestion
describe(buildArchiveGuidedChartSrc, () => {
```